python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -407,9 +410,22 @@ class RandomForestClassifier(Base):
         return treelite_handle
 
     def _get_protobuf_bytes(self):
-        fit_mod_ptr = self._obtain_treelite_handle()
+        if self.concat_handle and len(self.concat_model_bytes) > 0:
```

Comment:
that was the case initially but because of that the code would run the `save_model` function multiple times. 
Would it be okay if we only return model_protobuf_bytes but checked if the `self.concat_handle` is set or not?